charts/kyuubi/templates/kyuubi-service.yaml

```diff
@@ -15,6 +15,23 @@
   limitations under the License.
 */}}
 
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ $.Release.Name }}-headless
+  labels:
+    {{- include "kyuubi.labels" $ | nindent 4 }}
+spec:
+  type: ClusterIP
```

Comment:
we should add `clusterIp: None` here, as kubernetes doc said:

>  In this case, you can create what are termed headless Services, by explicitly specifying "None" for the cluster IP address (.spec.clusterIP).

Reply:
thanks to point that, I have to delete the headless svc to fix this.
luckily, there is no direct use of this headless service at the moment.